### ISSUE TITLE
[Firefox] Signed addon

### ISSUE THREAD

**Issue by BradyThe** [pixeltris/TwitchAdBlock#2]
Thank you for this extension. Below you can find it as signed Firefox addon.

Cheers

[twitchadblock-1.0-an+fx.xpi.zip](https://github.com/pixeltris/TwitchAdBlock/files/1590044/twitchadblock-1.0-an.fx.xpi.zip)

_Edit: The validation process returned one warning. "Unsafe assignment to innerHTML in redirect.js line 41 column 2", just fyi._

**Comment by pixeltris** [comment]
Thanks that works great. I'll put that xpi into the firefox folder and edit the readme to direct users to download it and drag it into firefox.

Yea I think that error comes from extensions not wanting you to do things like dynamic code execution? The original chrome version using "new Function(js)();" might be safer? I couldn't get it working in firefox so I just ran with injecting the modified playerjs as a new script tag. If you know of a way which wont throw any errors / warnings that would be good.

**Comment by BradyThe** [comment]
> If you know of a way which wont throw any errors / warnings that would be good.

Unfortunately I do not know anything about addon creation. I am just a bum who likes convenience. :-) I looked it up out of curiosity when I stumbled across the warning and found [this](https://developer.mozilla.org/en-US/docs/Archive/B2G_OS/Security/Security_Automation), but I have no use for it. It's just a warning and not an error, maybe it's the exception to the rule.

Thanks again. 👍
